.simplecov

```diff
@@ -1,5 +1,4 @@
 #!/bin/env ruby
-# frozen_string_literal: true
```

Comment:
Do we need to add this exception to the config?

Reply:
yes! done.